docs/examples/AccelerometerExample/src/AccelerometerExample.ls

```diff
@@ -59,10 +68,13 @@ package
             sprite.y = Math.clamp(sprite.y, 0, stage.stageHeight);
         }
 
-        protected function onAccelerometerData(x:Number, y:Number, z:Number) 
+        protected function onAccelerometerData(type:MobileSensorType, x:Number, y:Number, z:Number)
         {
+            if (type != MobileSensorType.Accelerometer)
+                return;
+
             directionX = x;
-            directionY = -y; 
+            directionY = -y;
```

Comment:
I wonder if there's any consistency in coordinate systems across mobile platforms (Android, iOS, WinPhone?). After we define the coord system that makes the most sense, we should look at this first so we don't end up double flipping and stuff :)